ChangeLog.md

```diff
@@ -33,6 +33,8 @@ See docs/process.md for more on how version tagging works.
   `PATH.basename("a/b/..")` returns `".."` instead of `"a"`. This is in line with
   the behaviour of both node and coreutils, and is already the case when using
   NODERAWFS". (#23180)
+- The factory function exposed in `-sMODULARIZE` mode is now marked as `async`
+  when `WASM_ASYNC_COMPILATION` is enabled (the default). (#23157)
```

Comment:
```suggestion
  when `WASM_ASYNC_COMPILATION` is enabled (the default). This may also
  allow run() to execute earlier. (#23157)
```

Reply:
My suggestion here isn't quite right, but I think it is worth mentioning the other issue (affecting `test_unicode_js..`) because having a mention in the changelog could help users that find weird startup behavior after this.

Reply:
Done